flepimop/gempyor_pkg/src/gempyor/calibrate.py

```diff
@@ -1,3 +1,27 @@
+"""
+calibrate.py
+
+Facilitates the calibration of the model using the `emcee` MCMC sampler. 
+Provides CLI options for users to specify simulation paramters. 
+
+Functions:
+    calibrate: Reads a configuration file for simulation settings.
+
+Options:
+   --config, -c             Path to the configuration file. 
+   --project_path, -p       Path to the flepiMoP project directory.  
+   --nwalkers, -n           Number of walkers (MCMC chains) to use. Overrides value in config.
+   --niterations            Number of MCMC iterations. Overrides value in config.
+   --nsamples               Number of samples to retain. OVerrides value in config.
+   --nthin                  Number of samples to thin. Overrides value in config.
+   --j, --jobs              Number of CPU cores to use for prallelization.
+   --id                     Unique ID for the run. 
+   --prefix                 Prefix for output files.
+   --resume                 Flag determining whether or not to resume the current calibration.
+   --resume_location, -r    Path to the location to resume the run from.
```

Comment:
This doesn't need to be in the module level documentation, this only applies to the CLI.